sdk/digitaltwins/Azure.DigitalTwins.Core/tests/AdtQueryBuilderTests/AdtQueryBuilderTests.cs

```diff
@@ -99,7 +98,8 @@ public void AdtQueryBuilder_Where_Comparison()
             new AdtQueryBuilder()
                 .Select("*")
                 .From(AdtCollection.DigitalTwins)
-                .Where("Temperature", QueryComparisonOperator.GreaterOrEqual, "50")
+                .Where()
+                .Comparison("Temperature", QueryComparisonOperator.GreaterOrEqual, "50")
```

Comment:
Temperature isn't a number? Why do we need to wrap it in quotation marks?

Reply:
Just pushed a change that can handle user inputted single quotes around both strings and numbers. It works but I'm not super sure about it, could you take a look?